config/settings/wdb.q

```diff
@@ -58,7 +63,7 @@ tpcheckcycles:0W
 
 // Server connection details
 \d .servers
-CONNECTIONS:`hdb`tickerplant`rdb`gateway`sort                                               // list of connections to make at start up
+CONNECTIONS:`hdb`tickerplant`rdb`gateway`sort`idb                                           // list of connections to make at start up
```

Comment:
I think this should probably be driven from the variables above, e.g. .wdb.hdbtypes,.wdb.tickerplanttypes etc. 
(I know this was probably wrong in the original code- not something you have added!)

Reply:
Removed this line from code, was unnecessarily(because of what you just said).